### ISSUE TITLE
The FlatQuant adapting the LLaVA multimodal model.

### ISSUE THREAD

**Issue by jpf888** [ruikangliu/FlatQuant#22]
hi，The FlatQuant loss is not within a reasonable range when adapting the LLaVA multimodal model. How should I check the loss during the calibration stage and ensure it is within a reasonable range?"

I followed the method in this link“https://github.com/ruikangliu/FlatQuant/issues/19#issuecomment-2832924408”，   and disabled quantization. The training loss (MSE) improved and converged better, roughly around 1e-2, but the output is still garbled. 

Do you have any suggestions? Have you done any experiments with 3B-scale multimodal models?"

**Comment by jpf888** [comment]
During training, the MSE of the first 7 layers remains around 1e-2, but it gradually increases in the deeper layers, reaching about ～4 or ～ 5 at the final layer."
